src/Microsoft.ComponentDetection.Detectors/npm/NpmComponentDetectorWithRoots.cs

```diff
@@ -147,6 +148,10 @@ protected Task ProcessAllPackageJTokensAsync(IComponentStream componentStream, J
 
     protected void ProcessIndividualPackageJTokens(ISingleFileComponentRecorder singleFileComponentRecorder, JToken packageLockJToken, IEnumerable<IComponentStream> packageJsonComponentStream, bool skipValidation = false)
     {
+        var lockfileVersion = packageLockJToken.Value<int>("lockfileVersion");
+        using var lockfileVersionTelemetry =
+            new NpmLockfileVersionTelemetryRecord() { LockfileVersion = lockfileVersion };
```

Comment:
Is there a reason this is broken up into two lines

Reply:
nope, just the way my IDE auto formatted it